pkg/cloud/gcpprovider.go

```diff
@@ -318,6 +318,11 @@ func (gcp *GCP) ClusterInfo() (map[string]string, error) {
 		attribute = c.ClusterName
 	}
 
+	// Use a default name if none has been set until this point
+	if attribute == "" {
+		attribute = "GKE Cluster #1"
```

Comment:
Is this fallback cluster name distinct enough that it can be recognized as the fallback value when doing debugging with a user?

Reply:
The `GetClusterInfo()` is implemented for all providers; in both the [aws](https://github.com/kubecost/cost-model/blob/5a227f20f69d37df06c46734839f27b52f4ccbaf/pkg/cloud/awsprovider.go#L1278) and the [azure](https://github.com/kubecost/cost-model/blob/4b3f80128a0f42e8d6216482baa4ae28792e7c00/pkg/cloud/azureprovider.go#L1151) implementations the default cluster name is following the scheme `<PROVIDER> Cluster #1` so I set the same for consistency!

Reply:
I'm not sure about the multi-cluster edge case, I'll look into it :)

Reply:
Thanks for the references, that's what I was hoping! Sorry for the slow response here.